sys/auto_init/netif/auto_init_slipdev.c

```diff
@@ -36,7 +35,7 @@
  */
 #define SLIPDEV_STACKSIZE       (THREAD_STACKSIZE_DEFAULT)
 #ifndef SLIPDEV_PRIO
-#define SLIPDEV_PRIO            (GNRC_NETDEV_MAC_PRIO)
+#define SLIPDEV_PRIO            (GNRC_NETIF2_PRIO)
```

Comment:
With the embargo and the plans to not support any legacy driver belonging to "netif1", is there a reason to keep this called "netif2"?

Reply:
The embargo is against master. Not against the branch the embargo is up in the first place ;-). And the rename is [definitely planned](https://github.com/RIOT-OS/RIOT/projects/6#card-5336168) (but only after everything is merged and definitely not in this only marginally related PR) ;-).

Reply:
It just came to my mind because I saw it here, not because this PR would be the one where to do the renaming.

I'll test asap.